tools/st2ctl

```diff
@@ -122,6 +122,7 @@ function st2start(){
   else
     service mistral start
   fi
+  (cd /opt/stackstorm/static/webui && nohup python -m SimpleHTTPServer 80) &>> ${LOGFILE} &
```

Comment:
In any case, I think it would be better to use some other port than 80 to avoid potential conflict with an existing web server running on that port.